css/auth.css

```diff
@@ -0,0 +1,79 @@
+.full-container {
+  width: 100vw;
+  height: 100vh;
+}
+
+.left-container {
+  width: 40%;
+  background-color: var(--secondary);
+  justify-content: flex-end;
+  gap: var(--scale-6);
+  color: var(--white);
+}
+
+.right-container {
+  width: 60%;
+  gap: var(--scale-2);
+  background-color: white;
+  justify-content: center;
+  align-content: center;
+  flex-wrap: wrap;
```

Comment:
I don't think it's needed here.

Reply:
I thought so too but without this form was not getting centered.

Reply:
Talked with @MaruthiPothuganti and Fixed the issue.